docs/MODEL.md

```diff
@@ -23,15 +25,17 @@ erDiagram
     OWNERSHIP_ENTRY {
         int id PK
         int note_id FK
-        string owner "Hex encoded public key of the note owner at this point in time"
+        string[32] reserve_nft_id "Reserve NFT id used as the key for the signed data inserted into the ergo box avltree"
+        byte[] a "'a' value used in signature"
+        bigint z "'z' value used in signature"
     }
-    SIGNATURE {
+    DENOMINATION {
         int id PK
-        int note_id FK
+        int type "Type enum of the denomination, 0 = gold"
```

Comment:
would make sense to have a denomination for erg as well? so this is used polymorphically?